src/fido2/extensions.c

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 SoloKeys Developers
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+#include <stdint.h>
+#include <string.h>
+#include "extensions.h"
+#if 0
```

Comment:
these `#if 0` look weird... I wish we had a set of .patch files and could simply apply them to the upstream when updating.